azure/resource_groups/app/template.json

```diff
@@ -342,6 +346,9 @@
           },
           "webTestLocations": {
             "value": "[variables('azureAvailabilityTestLocations')]"
+          },
+          "enableAlerts": {
```

Comment:
Would it be better to name this variable `enableWebTestAlerts`?

Reply:
Sure, my reasoning was that this is a local parameter to the `availability_test` deployment. I've changed it to `enableWebTestAlerts`

Reply:
Cool, just makes it a bit more easy to follow